Add hot wallets details to single user scheme (#304)
BitcoinDesign/Guide: guide/private-key-management/single-user-schemes.md

By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.

> This scheme is ideal for those new to bitcoin who may find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups, hot or spending wallets which store private keys locally and who's security model does not require a manual backup, or be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.

Reviewer: ```suggestion
This scheme is ideal for those new to Bitcoin who may find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups, hot or spending wallets which store private keys locally and who's security model does not require a manual backup, or be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
```

Reviewer: Good detail to add. This sentence should be broken up a bit though to improve readability.

<img width="604" alt="Screenshot 2021-05-11 at 04 34 47" src="https://user-images.githubusercontent.com/183140/117749556-43b4a600-b212-11eb-8d93-524c8859f82d.png">

Author: Let me know what you think of the change :)

> Automatic cloud backups can be useful for a variety of user experience levels. New Bitcoin users will likely find this method more frictionless. For new users, manual backups may actually be less secure as the process is likely foreign to them. This can lead to writing down or storing the backup in an unsafe manner. More experienced users can also make use of this scheme.  Wallets that store private keys locally, also known as hot or spending wallets, likely don't need the security manual backups offer. This scheme may also bridge the gap between new and experienced users. A security model may start new users with an automatic cloud backup. Then over time shift users to more secure scheme as they accumulate more bitcoin.

Reviewer 2: ```suggestion
This scheme can be a good fit for wallets meant for storing smaller amounts of bitcoin. This includes wallets for novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds. Wallets using automatic cloud backups also work well for what experienced users might call *hot* or *spending* wallets. 
```

Reviewer 2: Saying the same thing with fewer words.

Author: Cheers for that :)

> The automatic cloud backup scheme gives users reasonable security against theft whilst providing low risk for self-inflicted losses and a more seamless user experience for spending wallets of which a manual backup may be unnecessary.

Reviewer 2: While appears to be the more common form.

I suggest no change to this sentence, it is repeated in a similar form for all the schemes, dealing simply with the trade-off between theft and self-inflicted loss.

Reviewer 2: Made a commit suggestion so will resolve this comment.

> The automatic cloud backup scheme gives users reasonable security against theft whilst providing low risk for self-inflicted losses and a more seamless user experience for spending wallets of which a manual backup may be unnecessary.

Reviewer 2: ```suggestion
The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses.
```

- When risk of self-inflicted loss is higher than loss from theft
- For beginners, or products with progressive security

> - For lower values

Reviewer 2: ```suggestion
```

> - For hot or spending wallets that store their private keys locally

Reviewer 2: ```suggestion
- For wallets with frequent transactions of smaller amounts
```
